.eslintrc.base.js

```diff
@@ -0,0 +1,67 @@
+const path = require('path');
```

Comment:
Base configuration was pulled out into its own file from `.eslintrc`, but there should be no changes with what was there before.